### DISCUSSION TITLE
Runtime import module error: Cannot import name "x509" from "cryptography" in AWS Lambda

### DISCUSSION THREAD

**Discussion by mani0525** [stride3d/stride#1640] [General]
I'm attempting to use AWS Lambda to run the Python function below:
`import json
import boto3
import os
from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient

def lambda_handler(event, context):
    s3 = boto3.client("s3")
    connect_str = os.getenv('CONNECT_STR')
    blob_service_client = BlobServiceClient.from_connection_string(connect_str)
    records = event["Records"]
    
    print("Received records from s3 to update" + str(len(records)))
    
    for record in records:
        key = record["s3"]["object"]["key"]
        bucket_name = record["s3"]["bucket"]["name"]
        print("Processing object with key:" + str(key))
        file = s3.get_object(Bucket=bucket_name, Key=key)
        itr = file["Body"].iter_chunks()
        azure_container_client = blob_service_client.get_container_client(container=bucket_name)
        azure_container_client.upload_blob(name=key, data=itr)
        print("Object " + key + " transferred to azure successfully")`
The following code is entered into the console (Windows Power Shell), and it downloads all the dependencies: target='Target Directory' in pip3 setup azure-storage-blob

The target directory, which includes lambda_function.py, was then compressed before being deployed to AWS Lambda.

But I encountered the following error:
`{
  "errorMessage": "Unable to import module 'lambda_function': cannot import name 'x509' from 'cryptography.hazmat.bindings._rust' (unknown location)",
  "errorType": "Runtime.ImportModuleError",
  "requestId": "ca956e23-0d79-4897-8cfd-a0d74bca0098",
  "stackTrace": []
}`
The Python 3.9 runtime the x86_64 architecture

On my local computer, I can successfully execute from cryptography. hazmat.bindings._rust import x509 without any issues.

What might the issue be? How do I fix it?

Thank you.

**Comment by manio143** [comment]
@mani0525 This is offtopic on the repo for Stride game engine.
